mne/channels/montage.py

```diff
@@ -738,6 +743,45 @@ def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
         fids = [fids[key] for key in ('nasion', 'lpa', 'rpa')]
         coord_frame = 'unknown'
 
+    elif bvct is not None:
+        if not all(x is None for x in (hsp, hpi, elp, point_names, fif, egi)):
+            raise ValueError('hsp, hpi, elp, point_names, fif, egi must all '
+                             'be None if bvct is not None.')
+        _check_fname(bvct, overwrite='read', must_exist=True)
+
+        root = ElementTree.parse(bvct).getroot()
+        sensors = root.find('CapTrakElectrodeList')
+
+        fids = {}
+        dig_ch_pos = {}
+
+        fid_name_map = {'Nasion': 'nasion', 'RPA': 'rpa', 'LPA': 'lpa'}
+
+        # CapTrak is natively in mm
+        scale = dict(mm=1, cm=1e-1, auto=1e-2, m=1e-2)
+        if unit not in scale:
+            raise ValueError("Unit needs to be one of %s, not %r" %
+                             (sorted(scale.keys()), unit))
+
+        for s in sensors:
+            name = s.find('Name').text
+            fid = name in fid_name_map
+            coordinates = np.array([float(s.find('X').text),
+                                    float(s.find('Y').text),
+                                    float(s.find('Z').text)])
+
+            coordinates *= scale[unit]
+
+            # Fiducials
+            if fid:
+                fid_name = fid_name_map[name]
+                fids[fid_name] = coordinates
+            # EEG Channels
+            else:
+                dig_ch_pos[name] = coordinates
+
+        fids = [fids[key] for key in ('nasion', 'lpa', 'rpa')]
+        coord_frame = 'unknown'
```

Comment:
What does this `coord_frame` actually stand for? I know exactly how the coordinate system for these coords is defined ... should I put something other than `'unknown'` here?

In brief, it's "RAS" orientation:
X-axis: from LPA through RPA (to **R**ight)
Y-axis: orthogonal to X-axis and through Nasion (to front / **A**nterior)
Z-axis: orthogonal to XY-plane and through the scalp (up / **S**uperior)

cc @massich

Reply:
In fact, it seems like the CapTrak coordinate system is equivalent to the Neuromag coordinate system that is internally called `coord_frame='head'`

I confirmed this with a short test 
1. read a BVCT file with `transform=False`, then do `aa = dig_montage.dig_ch_pos`
1. do `dig_montage.transform_to_head()` and then `bb = dig_montage.dig_ch_pos`
1. Confirm `aa == bb` for each key: Which is true, even using `np.testing.assert_array_equal` (so numerical precision is good)

Reply:
Yes the RAS you mention above is the same as the neuromag head coordinate frame (we just call this "head" in MNE)